tensorflow/c/experimental/filesystem/modular_filesystem.cc

```diff
@@ -162,9 +162,9 @@ bool ModularFileSystem::FilesExist(
   return result;
 }
 
-Status ModularFileSystem::GetChildren(
-    const std::string& dir,
-    std::vector<std::string>* result /*, TransactionToken* token */) {
+Status ModularFileSystem::GetChildren(const std::string& dir,
+                                      std::vector<std::string>* result,
+                                      TransactionToken* token) {
```

Comment:
For these methods `token` is an input argument, right? It should come before `result` which will hold the results